src/utils/seo.ts

```diff
@@ -127,6 +129,20 @@ export function createSEOConfig({
         name: 'viewport',
         content: 'width=device-width, initial-scale=1, shrink-to-fit=no',
       },
+      // ...getOpenGraphAlternateLocales(locale),
```

Comment:
nit: is this something we will use in the future?

Reply:
Yes but we need to implement `fb_locale` param handling because Facebook sends this param to fetch the content in another locale. Talked to Abdellatif and he said it's nice to have later.